### ISSUE TITLE
Make the sections "Where to next" and  "See also" sections consistent across topics.

### ISSUE THREAD

**Issue by chirp1** [OpenLiberty/docs#1304]
Information at the end of topics that point users to other resources are inconsistent. I believe we want to use only "Where to next", to be consistent with the guides, and "See also" when appropriate, but whatever these headers are we want the headers to be consistent among topics. However, here are some examples of variations that I have seen, as of the opening of this issue. (Some might be fixed by the time you read this issue.):
- This topic has "Development resources" instead of See also:: https://draft-openlibertyio.mybluemix.net/docs/ref/general/#development-mode.html
- In this one, I'd like to see "Authentication filters" in a See also section: https://draft-openlibertyio.mybluemix.net/docs/ref/general/#auth-filter.html
- Instead of "Secure your applications with Open Liberty" at the bottom, I would expect the title to be "Where to next": https://draft-openlibertyio.mybluemix.net/docs/ref/general/#authentication.html
- Instead of "Secure your applications with Open Liberty" at the bottom of this other topic, which is next in the nav,  I would expect to see "Where to next": https://draft-openlibertyio.mybluemix.net/docs/ref/general/#authorization.html
- I would put "For more information, see Executor Management." in a "See also" section for the following topic: https://draft-openlibertyio.mybluemix.net/docs/ref/general/#thread-pool-tuning.html
- This one has "What to do Next". Put guides into "Where to next" and topics in "See also": https://draft-openlibertyio.mybluemix.net/docs/ref/general/#building-resilient.html
- Continue through the topics in the nav and fix any of those as appropriate.

**Comment by dmuelle** [comment]
@chirp1 all the topics in this issue have been updated to consistently use `See also` for reference links and `Where to next` for links to OL guides & demos

- https://draft-openlibertyio.mybluemix.net/docs/ref/general/#auth-filter.html
- https://draft-openlibertyio.mybluemix.net/docs/ref/general/#development-mode.html
- https://draft-openlibertyio.mybluemix.net/docs/ref/general/#authentication.html
- https://draft-openlibertyio.mybluemix.net/docs/ref/general/#authorization.html
- https://draft-openlibertyio.mybluemix.net/docs/ref/general/#thread-pool-tuning.html
- https://draft-openlibertyio.mybluemix.net/docs/ref/general/#building-resilient.html

**Comment by Charlotte-Holt** [comment]
@chirp1 Looking back at this one, maybe it would be best to just use the **See also** section rather than having both? I think as long as we make it clear that a link to a guide will take you to a guide, then it could be cleaner to have only one related docs-type section. 

I'm not sure whether all of the bullet points under **See also** should have sentences or whether it's appropriate for some to just be links with the name of the doc. It could depend on how descriptive the link text is.

**Comment by dmuelle** [comment]
The only two remaining docs with "Where to next?" headings are [Development Mode](https://draft-openlibertyio.mybluemix.net/docs/20.0.0.9/development-mode.html) and the [Metrics reference list](https://draft-openlibertyio.mybluemix.net/docs/20.0.0.9/metrics-list.html). I've updated Development mode and @Charlotte-Holt will update the reference list. Leaving this issue open until Dev mode changes publish to production with 20.0.0.9

**Comment by chirp1** [comment]
@dmuelle @Charlotte-Holt 
Hi! I looked over the [See also](https://github.com/OpenLiberty/docs/wiki/Stylistic-guidelines#see-also-link-sections) updates and discussed a few points about them with Pam.
- Instead of "In some cases, you can include the same link in the body of the topic and in the See also section, particularly if the linked topic is closely related or contains direct supporting info for the topic that you are linking from." How about if we make definitive when you can link to a topic more than once in your topic? The linking would really be only if you link to a specific section in the body and the whole topic in "See also". The wording might be something like: "However, if in the body of your topic you link to a specific section of another topic, you can link to the entire other topic in "See also".
- For "Do not include any supporting/introductory text for the links, unless the title of the linked topic does not make it clear why the information is relevant.", don't allow a mix of bullets where some have text and a link and others have just a link. That structure makes the links messy. If you really want text, see if you can do something like hover text. And better yet if the hover text automatically pulls in the intro paragraph of the topic that is being linked to. Here's an example with hover text on the related links: https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/twlp_setup_env.html

**Comment by dmuelle** [comment]
@chirp1 @Charlotte-Holt - I've made these changes.

- made it defintive to only include a link in both the body and `See also` when you've just link to a section of another topic in the body and the entire topic in `See also`.
- Specified not to include introductory text with `See also` links. 

As to hover text- I did a little research and hover text in general is not natively supported in Antora or Asciidoc. It would have to be provided by some kind of macro or custom patch from the design team. Which we can ask about- however- I don't think there are really going to be that many instances where it's necessary. 

In cases where supporting text is necessary, the writer can just work the link into the body of the topic rather than putting it in `See also`. Thoughts?
